storage/maria/s3.cnf

```diff
@@ -1,12 +1,22 @@
+[mariadbd]
+#
+# Uncomment line to enable
+#
+#plugin-maturity = alpha
+
 [mariadb]
-plugin-load-add=ha_s3
+#
+# Uncomment line to enable
+#
+#plugin-load-add = ha_s3
```

Comment:
Basically this is the most important change, why to disable plugin load,? MDEV-22980 is not suggesting nowhere this change.

Reply:
Because otherwise the server will error as reported by @GeoffMontee in https://jira.mariadb.org/browse/MDEV-22980. With the inline docs in the config it will be much easier for users to grasp how it works. Once the plugin is out of alpha we can start enabling it normally and it will work.

Reply:
Make sense to me. Don't know why is MDEV-22980 closed as not a bug?

Reply:
Because the error is intentional. The use has installed a plugin, and the plugin fails to load. It's an error.

It's not a fatal error, it will not abort server startup, but an error nevertheless.

Reply:
@ottok once when plugin reaches stability the workaround in config file will not be needed, what also make sense, I didn't know the process.
I would suggest to rebase the PR without `plugin-maturity` and `plugin-load-add` and maybe to leave the current workaround in jira comments?